src/admin/vote/Add.tsx

```
@@ -0,0 +1,356 @@
+                          suggestedStudents.filter((s) => s.grade == c.grade)
+                            .length
+                        }
+                      </mdui-badge>
+                    </mdui-tab>
+                  ))}
+                <p />
+                {classes
+                  .sort((a, b) => a.grade - b.grade)
+                  .map((c) => (
+                    <mdui-tab-panel slot="panel" value={c.id}>
+                      <div
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)